### DISCUSSION TITLE
Collabora online - No connection -- Built-in CODE Server

### DISCUSSION THREAD

**Discussion by Menzaah** [nextcloud/all-in-one#2100] [Questions] [answered]
Hello everyone,

Since yesterday, I realized that Collabora no longer works. The Nextcloud Office suite is not a tool that we use daily but which is important to us all the same.
I know it worked, no config changes were made apart from updates to Nextcloud AIO and its containers, respecting good practices each time.
Regarding the installation and operation of the device:

- I use Nginx Proxy Manager as reverse-proxy: v2.9.18
I followed the doc perfectly: [https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md)

![NPM](https://user-images.githubusercontent.com/78805485/222668254-30dcfa8f-35de-4ac6-9995-d08ff6a80ffd.PNG)

![NPM-Config](https://user-images.githubusercontent.com/78805485/222668301-769dbe91-c922-475c-9b70-cf423e05f99b.PNG)

- Nextcloud Version: v4.5.0

![AIO Interface](https://user-images.githubusercontent.com/78805485/222667389-8483db45-44c0-4289-ab23-97794ef14a0b.PNG)

![Capture](https://user-images.githubusercontent.com/78805485/222668334-a936269d-3ef1-4b18-bef2-2f63f5980dee.PNG)

![Capture2](https://user-images.githubusercontent.com/78805485/222668361-9bcea9f6-d2b6-405a-8c37-f953c7f50132.PNG)

- OS: Ubuntu 22.04.1 LTS

Here are some screenshots as well as docker-compose config files and logs that I was able to recover

[docker-compose-NginxProxyManager.txt](https://github.com/nextcloud/all-in-one/files/10879951/docker-compose-NginxProxyManager.txt)
[docker-compose.txt](https://github.com/nextcloud/all-in-one/files/10879953/docker-compose.txt)
[nextcloud-collabora.log](https://github.com/nextcloud/all-in-one/files/10879961/nextcloud-collabora.log)

What I tried to do:

- A complete restart of Nextcloud AIO, first stopping the dockers and then finally the mastercontainer
- Restart the dockers by deactivating Collabora, stop then restart again by reactivating Collabora
- Check that the "COLLABORA_SECCOMP_DISABLED=true" flag is present in my configuration

I did a lot of research before deciding to open this discussion here and I was able to find a "similar" problem reported as a bug on the NginxProxyManager github: [https://github.com/NginxProxyManager/nginx-proxy-manager/issues/1881](url)

Currently at an impasse and out of ideas, thank you for any help!

**Comment by szaimen** [comment]
Hi, can you follow https://github.com/nextcloud/all-in-one/discussions/1358?

**Reply by Menzaah** [reply]
Hello,

First of all thank you for your return, I did not expect to receive a return so quickly. It actually looks like a very good lead. Unfortunately after reading and re-reading, I can't find where to apply the change for "Allow list for WOPI requests" , yet I think I'm in the right place with the URL...
Am I doing something wrong or misunderstanding something ?

![Richdocuments](https://user-images.githubusercontent.com/78805485/222682489-f74dcaa0-4eb4-4e28-8a95-e21bc0f5aba6.PNG)

**Comment by szaimen** [comment]
> Am I doing something wrong or misunderstanding something ?

yes, you are misunderstanding. Use your own server was actually the correc option. Can you please run the mentioned commands on your server?

**Reply by szaimen** [reply]
![image](https://user-images.githubusercontent.com/42591237/222685687-c499a5ca-02cf-474a-94ed-23761ee278c5.png)

**Reply by Menzaah** [reply]
Yes, I understand what you mean, unfortunately when I want to validate by doing "Save" it switches to the other mode which you can see in my first answer.

![Richdocuments2](https://user-images.githubusercontent.com/78805485/222686294-3609e398-365a-444a-8c58-4af2717fa065.PNG)

Otherwise here are the commands requested and the result puzzles me because I don't see why it can't connect to port 443 overnight.

![Shell](https://user-images.githubusercontent.com/78805485/222686361-157edf1b-09e8-4661-9b44-8cb347ae281a.PNG)

**Comment by szaimen** [comment, marked as answer]
I see what is wrong. Please remove yourdomain.fr from the /etc/hosts file and restart AIO. That should make it work.

**Reply by Menzaah** [reply]
The timing is perfect, I was on file when you posted your answer. Indeed that was it, it works perfectly again. I'll keep going around but there's no reason.
Thank you for your help !

![Richdocuments3](https://user-images.githubusercontent.com/78805485/222690933-3d4049f3-9fa7-4db5-a612-1a601528d7d8.PNG)

**Comment by navigante** [comment]
The same problem exists with TALK High-performance backend server - not possible to connect, when domain is defined in /etc/hosts. 
Removing a domain entry resolve the issue, as proposed by @szaimen.
